pages/index.js

```diff
@@ -1,7 +1,90 @@
+import { outivities } from "@/lib/data";
+import Image from "next/image";
+import styled from "styled-components";
+import Head from "next/head";
+
 export default function HomePage() {
+  if (!outivities) {
+    return (
+      <>
+        <main>
+          <h1>Ooops... </h1>
+          <p>
+            Error Loading Outivities. We were unable to load the Outivities
+            list. Please try again later.
+          </p>
+        </main>
+      </>
+    );
+  }
```

Comment:
We still have kind of an error handling here as it was part of our tasks of the User Story.
As we are directly importing it from a file we are quite safe here but we are not sure we can fully drop this for now?

Reply:
Yes you can fully drop this 🚀